mtp_cashbook/apps/mtp_auth/models.py

```diff
@@ -0,0 +1,29 @@
+class MtpUser(object):
+    """
+    Authenticated user, similar to the Django one.
+
+    The built-in Django `AbstractBaseUser` sadly depends on a few tables and
+    cannot be used without a datbase so we had to create a custom one.
```

Comment:
This stuff would make perfect material for a commit message rather than a comment, if this were done in multiple discreet commits :wink: